test/testUtils/preparePassport.ts

```diff
@@ -93,6 +93,10 @@ export const preparePassport = (
     jwtSecret: private_pem_secret,
     name: LOGIN_STRATEGY,
     rpcUrl: `http://localhost:8544`,
+    jwtSignOptions: {
+      algorithm: 'RS256',
+      noTimestamp: true,
```

Comment:
Why `noTimestamp: true`?

Reply:
@nichonien pinging this

Reply:
@jrhender Since this is just a test case scenario, there was no requirement for validation of `timestamp`